DuckDuckGo/AppDependencyProvider.swift

```diff
@@ -22,6 +22,7 @@ import Core
 import BrowserServicesKit
 import DDGSync
 import Bookmarks
+import Subscription
```

Comment:
Does this need to be inside `#if SUBSCRIPTION` block?

Reply:
I think we should not, the imported class basically implements the runtime check equivalent to how we had previously used #if SUBSCRIPTION.
Plus for the release all targets/configuration will have the SUBSCRIPTION flag added (on macOS they already have, on iOS we only lack it in Release build config. atm). 
Plus it is likely that we will consider a cleanup of the DBP/NETWORK_PROTECTION/SUBSCRIPTION compile time feature flags some time after the launch.